src/ui/src/js/configs/routes.js

```diff
@@ -49,31 +49,23 @@ export default function routeConfig(
             },
           ],
           systems: [
-            '$rootScope',
-            'SystemService',
-            ($rootScope, SystemService) => {
-              return SystemService.getSystems().then(
-                  (response) => {
-                    $rootScope.sysResponse = response;
-                    $rootScope.systems = response.data;
-                  },
-                  (response) => {
-                    $rootScope.sysResponse = response;
-                    $rootScope.systems = [];
-                  },
-              );
-            },
-          ],
-          gardens: [
             '$rootScope',
             'GardenService',
             ($rootScope, GardenService) => {
               return GardenService.getGardens().then(
                   (response) => {
                     $rootScope.gardens = response.data;
+                    const localGarden = $rootScope.gardens.find(
+                        (item) => item.connection_type === 'LOCAL',
+                    );
+                    $rootScope.systems = localGarden.systems;
+                    response.data = localGarden.systems;
```

Comment:
I think these changes work, but the stuff going on here to populate `$rootScope.sysResponse` seems kinda sketchy.  I would try to track down what's using `sysResponse` and see if it's feasibly to make it just work with `rootScope.systems` rather than the response object.

If it's really impossible to detangle these things (I don't think it should be), then we at least need a comment here explaining why we're manipulating the response in this manner.